### PR TITLE
Fix/qcdata

### PR DESCRIPTION
**Pull request by wpfff**
Use get_parameter_data to get dataset data.
also don't use sqlite_base module anymore.

Still to be done: avoid usage of sqlite methods, and only rely on high-level/public imports.

### CODE REVIEW

File: plottr/data/datadict.py [hunk 1]
```diff
@@ -732,7 +732,12 @@ def add_data(self, **kw: Sequence):
         """
         dd = self.structure(same_type=True)
         for k, v in kw.items():
-            dd[k]['values'] = v
+            if isinstance(v, list):
```
Comment (astafan8) [inline plottr/data/datadict.py:735]: this change is solving the same issue as https://github.com/data-plottr/plottr/pull/68 , right?

File: plottr/data/qcodes_dataset.py [hunk 2]
```diff
@@ -139,56 +143,55 @@ def get_runs_from_db_as_dataframe(path, *arg, **kw):
...
+              value: DataDict containing that dependent and its
+                     axes.
     """
-    names = [n for n, v in ds.paramspecs.items()]
-    return {n : np.squeeze(ds.get_data(n, start=start, end=end)) for n in names}
+    ret = {}
+    pdata = ds.get_parameter_data()
+    for p, spec in ds.paramspecs.items():
+        if spec.depends_on != '':
+            axes = spec.depends_on.split(', ')
```
Comment (astafan8) [inline plottr/data/qcodes_dataset.py]: if we say that plottr works only with qcodes version 0.2.0+ (released ~7months ago) then this string-manipulation can be substituted with `spec.depends_on_` (with underscore) which returns a list of parameter names (as opposed to a comma-separated list of parameter names).

Comment (wpfff) [reply plottr/data/qcodes_dataset.py]: ah, thanks! i didn't know that function. will change that!

Comment (wpfff) [reply plottr/data/qcodes_dataset.py]: (i always wondered why the string format)

File: plottr/data/datadict.py (hunk 1 above)
Comment (wpfff) [reply plottr/data/datadict.py:735]: no, that's a different issue that #68 fixed -- but that issue is solved anyway by using get_parameter_data.

**Comment by wpfff** [comment]
@astafan8 -- @damazter was able to plot again in the wild. can you check if you find anything you're opposed to in my changes? :)

**Review by astafan8 — APPROVED** [review]

**Review by astafan8 — APPROVED** [review]
Good enough!

as discussed before, I'll follow up with a PR that removes the usage of `qcodes.dataset.sqlite.*`

**Pull request merged**